src/LuaPrinter.ts

```diff
@@ -30,8 +30,8 @@ export class LuaPrinter {
         [tstl.SyntaxKind.BitwiseAndOperator]: "&",
         [tstl.SyntaxKind.BitwiseOrOperator]: "|",
         [tstl.SyntaxKind.BitwiseExclusiveOrOperator]: "~",
-        [tstl.SyntaxKind.BitwiseRightShiftOperator]: ">>",
         [tstl.SyntaxKind.BitwiseArithmeticRightShift]: ">>>",
```

Comment:
Might be good to remove `BitwiseArithmeticRightShift` from AST, since it's not really a valid token

Reply:
Seconded - it doesn't make sense to have AST kinds for things that don't exist in lua.